### ISSUE TITLE
Small discrepancy between my generated sdc_aug and your youtube video

### ISSUE THREAD

**Issue by kwea123** [NVIDIA/semantic-segmentation#27]
I generated sdc_aug using the scripts and the models in this repo, but got small difference from what you show in youtube video.

This is what I got (aachen_000007_000022_gtFine_color):
![aachen_000007_000022_gtFine_color](https://user-images.githubusercontent.com/11364490/65014060-5650dc00-d958-11e9-8359-417d52556fab.png)
but it looks slightly different from [here](https://www.youtube.com/watch?v=aEbXjGZDZSQ&t=157), 3 frames after the GT. From what I am able to spot, the left black part, and the upper part of the right pole are different.

Did you apply any sort of post-processing on sdc_aug? I'm confused because I got different result from yours and my sdc_aug seemed generating very distorted images when the propagation is long. For example a +5 propagation (aachen_000005_000024_gtFine_color):
![aachen_000005_000024_gtFine_color](https://user-images.githubusercontent.com/11364490/65014245-feff3b80-d958-11e9-9f6d-09e4dc5a2d76.png)

**Comment by kwea123** [comment]
This is also different from the image in the paper:
![aachen_000008_000018_gtFine_color](https://user-images.githubusercontent.com/11364490/65014897-f3147900-d95a-11e9-9ae9-34b28cdf0341.png)

**Comment by bryanyzhu** [comment]
1. We retrain the sdc model and release the new one, so there is small discrepancy between your results and the figures in paper or youtube video. But I have done test before. Training on either augmented dataset gives similar results. 

2. Yes, long propagation will result in distorted images, and that's why we need label relaxation to control the damage. Your generated image looks good to me. 

The thing is, the noisy generated training samples will enhance your model's robustness. Although you may not see big improvements on validation set, it will generalize well to other scenes/domains. We may put a demo video soon on our Github page and you will see how our model can generalize to google street view images, foggy scenes, etc.

**Comment by kwea123** [comment]
Thanks for the confirmation. Looking forward to seeing the generalization demo!
